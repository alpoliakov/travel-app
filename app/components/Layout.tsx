import Head from 'next/head';
import React from 'react';

import Footer from './Footer';
import Header from './Header';
import { Widgets } from './Widgets/Widgets';

export default function Layout({ children }) {
  return (
    <div className="container">
      <Head>
        <meta charSet="utf-8" />
        <meta httpEquiv="Content-Type" content="text/html; charset=utf-8" />
      </Head>
      <Header />
      {/* widgets will be added to country-page and will be removed from here */}
      <Widgets /> 
      <main className="main">{children}</main>
      <Footer />
    </div>
  );
}
