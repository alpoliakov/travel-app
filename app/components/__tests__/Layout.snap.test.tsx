import React from 'react';
import renderer from 'react-test-renderer';

import Layout from '../Layout';

const MockChildren = <h1>Test</h1>;

it('Layout render correctly', () => {
  const tree = renderer.create(<Layout>{MockChildren}</Layout>).toJSON();

  expect(tree).toMatchSnapshot();
});
