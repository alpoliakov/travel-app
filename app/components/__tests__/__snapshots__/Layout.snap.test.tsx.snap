// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout render correctly 1`] = `
Array [
  <div
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "position": "fixed",
        "zIndex": 9999,
      }
    }
  />,
  <h1>
    Test
  </h1>,
]
`;
